api/schema/aws_account_resource.go

```diff
@@ -62,7 +65,7 @@ var (
 						},
 					},
 					attributeWithMode: attributeWithMode{
-						Mode: ImmutableAttributeMode,
+						Mode: WriteOnlyOnceAttributeMode,
```

Comment:
Revert those changes from `ImmutableAttributeMode` to `WriteOnlyOnceAttributeMode`.
`ImmutableAttributeMode` is the correct mode here.
`WriteOnlyOnceAttributeMode` is meant to be used for immutable secrets set by the client, which should never be returned by the server. This is not the case here.
```suggestion
						Mode: ImmutableAttributeMode,
```

Reply:
same as stated above. master account id isnt returned by the integrations read/update apis. hence made it writeonce.

Reply:
Immutable attribute is asking it to be sent in response of update and read. 
and role_arn, external_id.
accessMode cause we cannot update it as of now.

Reply:
Yes, this should be included in responses.
Revert this change.

Reply:
I asked you to remove the master account ID (https://github.com/illumio/terraform-provider-illumio-cloudsecure/pull/81#discussion_r1775776454), so it doesn't matter whether the backend APIs return that attribute or not.